{"name": "Divide the Two number -- @1.2 ", "status": "broken", "statusDetails": {"message": "AttributeError: 'Context' object has no attribute 'first_number'\n", "trace": "  File \"C:\\Users\\Jatin-PC\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\Jatin-PC\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"..\\steps\\steps.py\", line 50, in step_impl\n    if int(context.first_number) == int(context.second_number):\n  File \"C:\\Users\\Jatin-PC\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\behave\\runner.py\", line 321, in __getattr__\n    raise AttributeError(msg)\n"}, "steps": [{"name": "Given take the \"2\" and \"0\"", "status": "passed", "start": 1685414063766, "stop": 1685414063766}, {"name": "When verify if greater number between \"2\" and \"0\"", "status": "passed", "start": 1685414063766, "stop": 1685414063767}, {"name": "And divide greater with lower", "status": "passed", "start": 1685414063767, "stop": 1685414063768}, {"name": "Then Print the total result", "status": "passed", "start": 1685414063768, "stop": 1685414063769}, {"name": "And if both are same, return 0", "status": "broken", "statusDetails": {"message": "AttributeError: 'Context' object has no attribute 'first_number'\n", "trace": "  File \"C:\\Users\\Jatin-PC\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\Jatin-PC\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"..\\steps\\steps.py\", line 50, in step_impl\n    if int(context.first_number) == int(context.second_number):\n  File \"C:\\Users\\Jatin-PC\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\behave\\runner.py\", line 321, in __getattr__\n    raise AttributeError(msg)\n"}, "start": 1685414063769, "stop": 1685414063771}], "parameters": [{"name": "first_number", "value": "2"}, {"name": "second_number", "value": "0"}], "start": 1685414063764, "stop": 1685414063771, "uuid": "3829955d-4c6d-46ea-8b5f-bae60e33cfb0", "historyId": "174f701dff05850c1999e0ea5cb001bc", "fullName": "Perform an division Feature: Divide the Two number", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Perform an division Feature"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}